.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: AREA-CI
+
+on:
+  push:
+    branches-ignore:
+      - "ga-ignore-*"
+  pull_request:
+    branches-ignore:
+      - "ga-ignore-*"
+
+jobs:
+  run_front_tests:
+    runs-on: ubuntu-latest
+    container:
+      image: epitechcontent/epitest-docker:latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install dependencies
+        run: |
+          cd react-frontend
+          npm install
+      - name: Execute the tests
+        run: |
+          cd react-frontend
+          npm test a
```

Comment:
So based on my last comment, this steps should be in an other yml file like runTests.yml or checkCompile.yml.